# Generated by Django 4.0.5 on 2022-06-13 06:39

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Debtor',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=100)),
                ('aadharnumber', models.CharField(error_messages={'unique': '*Aadhar Number is already exists.'}, max_length=100, unique=True)),
                ('mobilenumber', models.CharField(error_messages={'unique': '*Phone Number is already exists.'}, max_length=10, unique=True)),
                ('profile', models.ImageField(default='profile.png', upload_to='media/')),
                ('created_at', models.DateTimeField(auto_now_add=True, null=True)),
                ('updated_at', models.DateTimeField(auto_now=True, null=True)),
                ('user_id', models.CharField(blank=True, max_length=500)),
                ('pancard', models.CharField(error_messages={'unique': '*Pan Number is already exists.'}, max_length=100, unique=True)),
                ('rationcard', models.CharField(error_messages={'unique': '*Ration Card Number is already exists.'}, max_length=100, unique=True)),
                ('address', models.TextField(max_length=1000)),
                ('email', models.EmailField(error_messages={'unique': '*Email Id is already taken.'}, max_length=254, unique=True)),
                ('first_name', models.CharField(max_length=100)),
                ('last_name', models.CharField(max_length=100)),
                ('place', models.CharField(max_length=100)),
            ],
        ),
    ]
